theme/templates/resource-landing-page/modals.html

```diff
@@ -1393,6 +1397,60 @@ <h4 class="modal-title" id="add-relation"> Add Relation </h4>
         </div>
     </div>
 
+    <div class="modal fade" id="customize-citation-dialog" tabindex="-1" role="dialog" aria-labelledby="customize-citation"
+         aria-hidden="true">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                {% if custom_citation %}
+                    <form action="/hsapi/_internal/{{ cm.short_id }}/citation/{{ citation_id }}/update-metadata/" method="POST"
```

Comment:
I was right about the citation id being used for delete.  A resource can only have one citation, so we really don't need the citation id.

Reply:
Sure. I was following the pattern established in the codebase. Can you confirm the hsapi/_internal call allows for posts to conditionally omit supply citation_id ? Otherwise, I'd rather follow established conventions and refactor the endpoint and calls to it in the future.

Reply:
ah, I see the established pattern now.  You're right, just use the citation id.  I thought you might have added an endpoint for deleting the citation, but I see now you're just using the metadata endpoints, which makes sense.